### PR TITLE
Fix webm video seeking issue

### LINKED ISSUE #33: Envision `Record` Video not Seek-able
The resulting recorded video from envision is not seek-able when using most video players. This makes the video much less useful.

RecordRTC documentation would suggest that `getSeekableBlob` would fix this issue: https://github.com/muaz-khan/RecordRTC/blob/master/README.md#global-apis

### PR DESCRIPTION
**Pull request by liamchzh**
close #33 

According to the [README](https://github.com/muaz-khan/RecordRTC/blob/master/README.md#global-apis), RecordRTC documentation suggests using getSeekableBlob to fix this issue.

### CODE REVIEW

File: envision/web/src/index.html [hunk 1]
```diff
@@ -7,6 +7,9 @@
       content="width=device-width, initial-scale=1, shrink-to-fit=no"
     />
     <title>Envision</title>
+
+    <!-- link this EBML.js to fix video seeking issues -->
+    <script src="https://www.webrtc-experiment.com/EBML.js"></script>
```
Comment (liamchzh) [inline envision/web/src/index.html:12]: The tricky thing is using the `getSeekableBlob` required to link this EBML.js file. I didn't see any documentation mentioning this but I saw an error in the console log.

**Comment by liamchzh** [comment]
@Gamenot Thanks for providing the solution to this issue.

File: envision/web/src/index.html (hunk 1 above)
Comment (aurorahcx) [reply envision/web/src/index.html:12]: It mentions on line 1904-1907 in https://recordrtc.org/RecordRTC.js.html. 
"EBML.js copyrights goes to: https://github.com/legokichi/ts-ebml"

Comment (aurorahcx) [reply envision/web/src/index.html:12]: It mentions on line 1904-1907 of https://recordrtc.org/RecordRTC.js.html
"EBML.js copyrights goes to: https://github.com/legokichi/ts-ebml"

**Review by aurorahcx — APPROVED** [review]

File: envision/web/src/index.html (hunk 1 above)
Comment (Gamenot) [reply envision/web/src/index.html:12]: It looks OK, they have MIT license.

**Review by Gamenot — APPROVED** [review]

**Pull request merged**